spec/README.md

```diff
@@ -137,23 +121,13 @@ based on this version being invalid.
 
 > For example, existing interfaces or fields will not be removed.
 
-Future versions of this specification will not introduce syntax that cannot be
-represented by this version of the data model.
-
-> For example, a future version could introduce a new keyword.
-> The future version's data model would provide an interface for that keyword
-> while this version of the data model would parse the value into
-> the interface `UnsupportedStatement`.
-> Both data models would be "valid" in their context,
-> but this version's would be missing any functionality for the new statement type.
-
 > [!IMPORTANT]
 > This stability policy allows any of the following, non-exhaustive list, of changes
 > in future versions of this specification:
+> - Future versions may define new syntax and structures
+>   that would not be supported by this version of the specification.
 > - Future versions may add additional structure or meaning to existing syntax.
 > - Future versions may define new keywords.
-> - Future versions may define annotations that use portions of the `reserved-annotation`
->   syntax.
 > - Future versions may make previously invalid messages valid.
 > - Future versions may define additional functions in the default registry
 >   or may reserve the names of functions for the purposes of interoperability.
```

Comment:
Unicode specifications allow deprecation of a particular entity. However, deprecation does **not** mean removal; conformation implementations can (and sometimes should) continue to support that entity, but it is marked as being discouraged, eg resulting in lint warnings.

Reply:
@macchiati Do you have a suggestion here?

I think we say that we might deprecate (but never remove) elsewhere?

Reply:
Perhaps something like the following. It doesn't have to be here.

```suggestion
>   or may reserve the names of functions for the purposes of interoperability.
> - Future versions may deprecate elements (keywords, functions, options, etc.) but may not remove them. (Deprecation means that they are strongly discouraged; for example, linting implementations may issue warnings.)
```

Reply:
A few lines down we already have this:
> Future versions may deprecate functions, options, or option values.

Is that not sufficient?